scwx-qt/res/qt6ct_colors/airy.conf

```diff
@@ -0,0 +1,4 @@
+[ColorScheme]
```

Comment:
Assuming the reason for pulling in the resources like this (as opposed to a git submodule) is to allow for modifications?

Reply:
Let me see about pulling it as a submodule. The issue is qt6ct is not really meant to be used as a library in this way. It provides a Qt Platform Abstraction (QPA) platform theme, which is not really useful to us because it is based system wide binary files. I am just using the code which parses the color palette. I think one of the shared libraries it makes could be used. I just need to figure out how to link it using cmake.

Reply:
I have it working as a submodule.

Reply:
The only things that is not coming from the submodule are the palette files. I am not sure the best way to include them in the binary, My Linux solution would be to symlink the directory, but I remember symlinks not really working on Windows. If you have any suggestion, I would appropriate that.

Reply:
To reference the submodule configuration files, I expect you should be able to do something like the following:

```
<file>res/qt6ct_colors/airy.conf</file>
```